hw/mcu/stm/stm32l1xx/src/ext/Drivers/STM32L1xx_HAL_Driver/Src/stm32l1xx_hal_spi.c

```diff
@@ -170,7 +170,8 @@ static HAL_StatusTypeDef SPI_WaitOnFlagUntilTimeout(struct __SPI_HandleTypeDef *
 @endverbatim
   * @{
   */
-
+#if 0
+// BW : The weak direction is ignored by gcc, and so fails to pick up the proper implementation from stm32l1xx_hal_spi_ex.c

```

Comment:
C++ style comment should not be used.
Adding **BW :** is out of place

Reply:
Ok